bin/aursync

```diff
@@ -228,7 +229,7 @@ aurchain -t -- "$@" >pkginfo_0
   done
 
   if ((chkver)); then
-      cut -f1,3 pkginfo_0 | aurcheck -d "$repo" -r "$root" -c
+      cut -f1,3 pkginfo_0 | aurcheck -d "$repo" -r "$root"
```

Comment:
Drawing your attention here because I can't come up with a good way to test this, but I think it should work™